### ISSUE TITLE
cannot find file sharing in settings panel in docker desktop

### ISSUE THREAD

**Issue by py563** [docker/for-win#5851]
- [Yes] I have tried with the latest version of my channel (Stable or Edge)
  - [Yes] I have uploaded Diagnostics
  - Diagnostics ID: missing

### Expected behavior
should be able to see the below settings as mentioned in website 
![image](https://user-images.githubusercontent.com/6139284/75119450-01548700-56a9-11ea-8341-ad511c1ebe13.png)

### Actual behavior
![image](https://user-images.githubusercontent.com/6139284/75119513-7fb12900-56a9-11ea-9755-da076affefd3.png)

i get this error too 
```bash 
nginx_1    | 2020/02/23 20:26:51 [crit] 1#1: pread() "/etc/nginx/conf.d/site.conf" failed (21: Is a directory)
nginx_1    | nginx: [crit] pread() "/etc/nginx/conf.d/site.conf" failed (21: Is a directory)
```
**update for error it only comes when the code repo is on a sdcard**, i think due to no automount of external drives on wsl2

### Information

  - Windows Version: Win 10 Pro Insider Preview Build 19591.rs-pre-200102-1216
  - Docker Desktop Version:2.2.1.0(42746)
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: no

### Steps to reproduce the behavior

  1. I just installed docker desktop both edge and stable releases and found about this issue.
  2. I have wsl2 settings on the docker.
  3. I have the repository on a sdcard downloaded from git works fine on ubuntu vm on friend's machine

**Comment by mat007** [comment]
The Settings page you pasted from the documentation is for the LinuxVM backend implementation, what you’re seeing is correct for WSL 2: there is no need to configure the settings you’re not seeing as it’s automatic with WSL 2.

**Comment by py563** [comment]
I pulled the image from here on [docker desktop for windows page](https://hub.docker.com/editions/community/docker-ce-desktop-windows) and many blogposts about docker for windows had same image expect for at somewhere in docker webpage the new image did show up that is when I figured out UI Changed as I mentioned in earlier comment.

I was looking for that option because for the error i got everyone pointed out to that settings. Later i    figured it is got to do with code being on a sdcard, I knew both WSL and WSL2 does not auto mount external storage devices and I manually do it to work on vscode remote. So I after giving a try to move them to system partitions everything was fine. So, I just better error description or side note reminding people about that wsl restrictions to less tech savvy would solve this issue.
